packages/anon-creds/src/AnonCredsApi.ts

```diff
@@ -0,0 +1,20 @@
+import type { RegisterSchemaApiOptions, RegisterSchemaApiReturn } from './AnonCredsApiOptions'
+import type { AnonCredsSchema } from './models/vdr'
+
+export interface AnonCredsApi {
```

Comment:
Just removed the API and API options. Posting the contents of those files here for future reference.

**AnonCredsApi.ts**

```typescript
import type { RegisterSchemaApiOptions, RegisterSchemaApiReturn } from './AnonCredsApiOptions'
import type { AnonCredsSchema } from './models/registry'

export interface AnonCredsApi {
  getSchema(schemaId: string): Promise<AnonCredsSchema>
  register(options: RegisterSchemaApiOptions): Promise<RegisterSchemaApiReturn>

  // TODO: getCredentialDefinition
  // TODO: registerCredentialDefinition

  // NOTE: Do we want to expose these methods? Probably, but it should also just auto trigger when   is set to true in the credential definition.
  // TODO: getRevocationRegistryDefinition
  // TODO: registerRevocationRegistryDefinition

  // NOTE: Naming TDB
  // NOTE: Do we want to expose these methods? Probably, but it should also just auto trigger when credentials are revoked
  // NOTE: How will users revoke credentials? This hasn't been supported in the past, but is an important feature going forward that we need to implement.
  // TODO: getRevocationList
  // TODO: registerRevocationList
}
```

**AnonCredsApiOptions**

```typescript
import type { AnonCredsSchema } from './models/registry'

// FIXME: we should update the name in the AnonCreds registry interface to include extra wording in the name to prevent name clashes.
export interface RegisterSchemaApiOptions {
  schema: AnonCredsSchema

  // Identifier of the identifier that will create the schema. In most cases a did, but can also be an URI.
  //   TODO: do we want an issuerId? Or should we add such properties to an options object that is different per method
  // more like the did registration interface?
  issuerId: string
}

export interface RegisterSchemaApiReturn {
  schemaId: string
  schema: AnonCredsSchema
}
```